src/style/common/label.less

```diff
@@ -22,6 +22,11 @@
   font-weight: @font-weight-semibold;
 }
 
+.label-sm {
+  font-size: @font-size-small;
+  font-weight: @font-weight-semibold;
+}
+
```

Comment:
I think that's `label-2` in `typing.less` , the new label templates @przemvs  was talking about the other day